src/states/elder/mod.rs

```diff
@@ -1575,6 +1561,10 @@ impl Approved for Elder {
             });
 
             self.send_neighbour_infos();
+
+            if let Some(relocate_details) = self.chain.poll_relocation() {
+                self.vote_for_relocate(relocate_details)?;
+            }
```

Comment:
Why are we voting here in addition to when we process `Online`/`Offline`? Every `Online` and `Offline` will generate a new `SectionInfo` event, anyway (at least until we have proper promotions/demotions). Or is the purpose to enable relocating more than one node per churn event?

Reply:
If the new member is going to be immediately promoted to elder (in case of Online) or if the removed member is elder (in case of Offline), then we can't vote `Relocate` there because the `churn_in_progress` flag is set to true. That's why we have to vote in `handle_section_info` because the flag is set to false here. On the other hand, for non-elder members, we can vote directly in `handle_online` / `handle_offline`, although that will only happen when promotions/demotions are implemented.